README.md

```diff
@@ -2,5 +2,12 @@
 We've built a simple application that demonstrates how LaunchDarkly's SDK works.  Below, you'll find the basic build procedure, but for more comprehensive instructions, you can visit your [Quickstart page](https://app.launchdarkly.com/quickstart#/).
 ##### Build instructions  #####
 1. Open the ```hello-react-native``` directory
-2. Copy the mobile key from your account settings page from your LaunchDarkly dashboard into `App.js` 
-3. Run your application through the Emulator or on a real device.
\ No newline at end of file
+2. Copy the mobile key from your account settings page from your LaunchDarkly dashboard into `App.js`.
+3. Run `npm install start` in `hello-react-native`. This should download and create the directories for the project.
+4. Run `npm run start` to start the server to deliver the bundle url. Make sure this is running before you start your application. 
+
+##### iOS specific build instructions #####
+1. `cd` into `iOS` folder 
+2. run `pod update LaunchDarkly`
```

Comment:
Given that you're bumping the `LaunchDarkly` pod version below, I recommend that you do the pod install yourself and commit the changes. That way there is one less step for customers to have to run the sample app.

We generally shouldn't require customers to have to do a pod install. This was merely an omission on my part from when I bumped `hello-react-native` to v3 of the SDK.

Reply:
oh, I assumed we just didn't want to commit all of that to the repo. I'll do that and update.

Reply:
I added the podfile and changed the readme accordingly. Let me know if there is anything else outstanding @bwoskow-ld .